### ISSUE TITLE
[dataservice-portal] datasource 관련 오류 발생 (org.postgresql.util.PSQLException: FATAL: database "DATA" does not exist)

### ISSUE THREAD

**Issue by vanilet** [IoTKETI/citydatahub_data_marketplace_module#2]
메뉴얼을 참고하여 dataservice-portal를 실행 시 프로그램 실행시 datasource 관련 오류 발생하고 모듈이 죽음
1. dataservice-portal의 상세 로그
`
2022-11-25 15:12:14.014 ERROR [APP_NAME_IS_UNDEFINED,,] 9929 --- [DiscoveryClient-InstanceInfoReplicator-0] com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Exception during pool initialization.
org.postgresql.util.PSQLException: FATAL: database "DATA" does not exist
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2468)
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2587)
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:134)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:250)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:365)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:194)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:460)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:534)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at com.zaxxer.hikari.HikariDataSource$$FastClassBySpringCGLIB$$eeb1ae86.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:746)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:136)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:124)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:185)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:688)
	at com.zaxxer.hikari.HikariDataSource$$EnhancerBySpringCGLIB$$2513c525.getConnection(<generated>)
	at org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:151)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:115)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:78)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:324)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.getProduct(DataSourceHealthIndicator.java:122)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.doDataSourceHealthCheck(DataSourceHealthIndicator.java:109)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.doHealthCheck(DataSourceHealthIndicator.java:104)
	at org.springframework.boot.actuate.health.AbstractHealthIndicator.health(AbstractHealthIndicator.java:84)
	at org.springframework.boot.actuate.health.CompositeHealthIndicator.health(CompositeHealthIndicator.java:68)
	at org.springframework.cloud.netflix.eureka.EurekaHealthCheckHandler.getHealthStatus(EurekaHealthCheckHandler.java:103)
	at org.springframework.cloud.netflix.eureka.EurekaHealthCheckHandler.getStatus(EurekaHealthCheckHandler.java:99)
	at com.netflix.discovery.DiscoveryClient.refreshInstanceInfo(DiscoveryClient.java:1382)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:117)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
`

**Comment by vanilet** [comment]
확인 결과 현재 master 소스 트리의 src/main/resources/application.yml 파일 내용에 오류가 있음.

현재 master 소스 트리의 해당 파일 내용
```YAML
#common

server:
  port: 7930
  tomcat:
    max-connections: 2000 # 서버에서 처리할 최대 연결 수
    max-threads: 1000     # 서버에서 처리할 수 있는 최대 동시 요청 수
    accept-count: 1000    # 요청 쓰레드가 다 사용중인 경우 대기할 큐 수
  connection-timeout: 15000


spring:
  application:
    name: dh-dataservice-dataset-provider
  profiles:
    include:
    - env-old
    active: dev

logging:
  level:
    kr.co.n2m.smartcity: debug

springdoc:
  version: '1.0'
  api-docs:
    path: /api-docs
  default-consumes-media-type: application/json
  default-produces-media-type: application/json
  swagger-ui:
    operations-sorter: alpha
    tags-sorter: alpha
    path: /swagger-ui.html
    disable-swagger-default-url: true
    display-query-params-without-oauth2: true
  paths-to-match:
  - /dataservice/datasetCategories
  - /dataservice/datasetCategories/{datasetCategoryId}
  - /dataservice/dataset
  - /dataservice/dataset/{datasetId}
  - /dataservice/dataset/{datasetId}/columns
  - /dataservice/dataset/{datasetId}/columns/{columnId}
  - /dataservice/dataset/{datasetId}/instance
  - /dataservice/dataset/{datasetId}/instance/{instanceId}
  - /dataservice/dataset/{datasetId}/searchCondition
  - /dataservice/dataset/{datasetId}/searchCondition/{searchConditionId}
  - /dataservice/dataset/{datasetId}/wishlist
  - /dataservice/dataset/{datasetId}/wishlist/{wishlistId}
  - /dataservice/dataset/{datasetId}/requestUsage
  - /dataservice/dataset/{datasetId}/requestUsage/{requestUsageId}
  - /dataservice/dataset/{datasetId}/requestUsage/{requestUsageId}/receptions
  - /dataservice/dataset/{datasetId}/requestUsage/{requestUsageId}/receptions/{recvId}
  - /dataservice/dataset/{datasetId}/requestUsage/{requestUsageId}/payments
  - /dataservice/dataset/{datasetId}/requestUsage/{requestUsageId}/payments/{payId}
  - /dataservice/dataset/{datasetId}/data
```

**Comment by vanilet** [comment]
오류 원인 파악 결과, src/main/resources/application.yml에서 `spring.application.profiles.include`의 설정값이 `env-old`로 구성되어있음.

하지만, 실제 소스트리에서 src/main/resources를 살펴보면 `application-env-old.yml` 파일은 존재하지 않고 `application-env.yml` 파일만 존재함.

해당 설정값(`spring.application.profiles.include`)을 env로 설정하면 `src/main/resources/application-env.yml` 파일을 제대로 참고하여 정상 동작함.

해당 내용 소스 트리에 반영 필요함.

**Comment by vanilet** [comment]
resolved in commit 334220802e5361d88c0a10263764a2939f6b9998
